api/paidAction/zap.js

```diff
@@ -78,10 +91,14 @@ export async function onPaid ({ invoice, actIds }, { models, tx }) {
       SELECT COALESCE(SUM(msats), 0) as msats
       FROM forwardees
     ), recipients AS (
-      SELECT "userId", msats FROM forwardees
+      SELECT "userId", msats, msats AS "stackedMsats" FROM forwardees
       UNION
       SELECT ${itemAct.item.userId}::INTEGER as "userId",
-        ${itemAct.msats}::BIGINT - (SELECT msats FROM total_forwarded)::BIGINT as msats
+        CASE WHEN ${!!invoice?.invoiceForward}::BOOLEAN
+          THEN 0::BIGINT
+          ELSE ${itemAct.msats}::BIGINT - (SELECT msats FROM total_forwarded)::BIGINT
+        END as msats,
+        ${itemAct.msats}::BIGINT - (SELECT msats FROM total_forwarded)::BIGINT as "stackedMsats"
```

Comment:
We want `stackedMsats` to go up, but we don't want their balance to increase if it's a p2p zap.

Reply:
Makes sense but you added `recipients."stackedMsats"` but never use it since afaict, you don't need to.